frontend/Store.js

```diff
@@ -150,6 +151,7 @@ class Store extends EventEmitter {
     this._establishConnection();
     this._eventQueue = [];
     this._eventTimer = null;
+    this._emitSearchRootsChange = debounce(this.filterSearchRoots, 100);
```

Comment:
From the error message in [Travis build](https://travis-ci.org/facebook/react-devtools/builds/101731744), it seems you should declare that function on `Store` as an empty function.